packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/en.ftl

```diff
@@ -1,7 +1,8 @@
-passwordResetAccountRecovery-subject = Password updated using recovery key
-passwordResetAccountRecovery-title = Your account password was reset with a recovery key
-passwordResetAccountRecovery-description = You have successfully reset your password using a recovery key from the following device:
-passwordResetAccountRecovery-action = Create new recovery key
-passwordResetAccountRecovery-regen-required = You will need to generate a new recovery key.
+passwordResetAccountRecovery-subject-2 = Your password has been reset
+passwordResetAccountRecovery-title-2 = Password reset successfully
+passwordResetAccountRecovery-description-2 = You used your account recovery key to update your password from:
+passwordResetAccountRecovery-action-2 = Create a new account recovery code
+passwordResetAccountRecovery-regen-required-base = You’ll need to sign in again on all of your synced devices. Remember to create a new account recovery key to replace the one you used
```

Comment:
I think it'd be more straightforward for localizers to have this text in the mjml and txt strings, rather than calling the -base string.